packages/eslint-plugin/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.2.0 (Unreleased)
```

Comment:
```suggestion
## x.x.x (Unreleased)
```
❓ :)

We should update our docs as well to reflect it.

Reply:
@gziolo I thought we put actual version numbers to reflect via semver the type of change.  Since this is a new feature I'm assuming that's why @aduth you chose 2.2.0?

Reply:
I have no idea anymore.

Reply:
@nerrad - the issue is that with technical version bump for WordPress core releases we run in the rabbit hole where it is no longer reliable what you see in the changelog file :(

Reply:
hmm.  Then I'm no longer sure what we do either.  Should probably be brought up in the next core-js meeting then and maybe clarified in docs?  I did a (very) brief perusal of the docs and I don't see anything mentioned specific one way or another.  I know it's been my pattern (based on what I used to see) to add the version but can obviously change if that's no longer the way to do things :)

Reply:
The current guidelines live here:

https://github.com/WordPress/gutenberg/blob/master/packages/README.md#maintaining-changelogs

Which would align with my original change. It sounds as though they need to be updated. I'll make a note in the agenda for tomorrow's Core JS chat.